RELEASE.md

```diff
@@ -4,12 +4,22 @@
 
 * node v16.x.x (LTS)
 
-## How to deploy
+## How to run
+### Frontend
 
-Run following command in the folder with `index.html`:
+The built static files (SPA) are located in the `frontend` folder, you can serve it using your favorite server. For example:
+```sh
+npx serve frontend -s
+```
+(note the -s flag for SPA, that means all requests will return index.html)
 
+### Backend
+The backend is only necessary for network play, `cd` to the `backend` folder and run:
 ```sh
-npx serve .
+# install dependencies using npm
+npm install
```

Comment:
This is release documentation. U sure user have to run npm install?

Reply:
I mean we could probably include the backend dependencies, but it does not make much sense, since you still need node to run it.

At this point, we don't need to build the prod artifacts anyway (render.com build it itself, they don't support only deploying artifacts, currently), so I'm leaning towards removing creation of the zip altogether. After all, we're building a web app, not a desktop app, so users don't need to download the release. Let's discuss this later.